src/taoensso/encore.cljc

```diff
@@ -71,16 +71,17 @@
 
 ;;;; Core macros
 
-(defmacro compile-if
-  "Evaluates `test`. If it doesn't error and returns logical true, expands to
-  `then`, otherwise expands to `else`. Stolen from `clojure.core.reducers`.
-  (compile-if (Class/forName \"java.util.concurrent.ForkJoinTask\")
-    (do-cool-stuff-with-fork-join)
-    (fall-back-to-executor-services))"
-  [test then & [else]]
-  (if (try (eval test) (catch Throwable _ false))
-    `(do ~then)
-    `(do ~else)))
+#?(:clj
```

Comment:
I think these reader conditionals are required now to make sure they're only executed when reading in the Clojure context. Before with cljx, there were two separate files created for ClojureScript, and the macros were only evaluated in the Clojure compilation context.